File: cpp/bc2377.cpp
#include<iostream>

using namespace std;

int main()
{
    int nPe, kTo, dPe, vKi, vPe;
    
    cin >> kTo >> dPe >> vKi >> vPe;
    
    nPe = kTo / dPe;
    
    cout << kTo*vKi + nPe*vPe << endl;
}

File: python/bc2377.py
fVec = input().split(" ")
kTo = int(fVec[0])
dPe = int(fVec[1])

sVec = input().split(" ")
vKi = int(sVec[0])
vPe = int(sVec[1])

nPe = kTo // dPe

print(kTo*vKi + nPe*vPe)